packages/core/core/src/applyRuntimes.js

```diff
@@ -66,8 +69,23 @@ export default async function applyRuntimes({
         if (applied) {
           let runtimeAssets = Array.isArray(applied) ? applied : [applied];
           for (let {code, dependency, filePath, isEntry} of runtimeAssets) {
+            let sourceName = path.join(
+              path.dirname(filePath),
+              `runtime-${md5FromString(code)}.${bundle.type}`,
+            );
+
+            let sourcemap = SourceMap.generateEmptyMap({
```

Comment:
I'm probably forgetting something but why do runtimes need a source map that is just 1:1 with the code again?

Reply:
To have the source content and in case we do transformations on the runtime

Reply:
Ah because it's not actually on disk. Got it